camayoc/tests/qcs/cli/utils.py

```diff
@@ -164,10 +207,16 @@ def source_show(options, output, exitstatus=0):
 scan_restart = functools.partial(cli_command, 'qpc scan restart')
 """Run ``qpc scan restart`` command with ``options`` returning its output."""
 
-scan_add = functools.partial(cli_command, 'qpc scan add')
+scan_add_output = functools.partial(cli_command, 'qpc scan add')
```

Comment:
This should continue named scan_add and the helper function that performs some assertions should be renamed.

Reply:
@elyezer I have almost completed this update.  Should I do the same for the cred/source commands?  They also had similar issues.

Reply:
Yes please, my intention is to be consisted and have a set of raw tools to perform a basic CLI commands and then we can grow other layers on top of them so if we need more abstraction we use the higher levels but we may want a more raw cli call depending of the testing.

For example I may want to perform negative tests, or try something else in order to break and that is not predicted by the higher levels.